### PR TITLE
Fixes 366

### PR DESCRIPTION
**Pull request by ReidWilliams**
https://github.com/computablelabs/ffa-frontend/issues/366

### CODE REVIEW

File: src/components/ui/StaticFileMetadata.vue [hunk 1]
```diff
@@ -67,8 +69,14 @@ export default class StaticFileMetadata extends Vue {
     return !!this.ffaListing ? this.ffaListing.license : ''
   }
 
-  public get fileSize(): number {
-    return !!this.ffaListing ? this.ffaListing.size : 0
+  public get fileSize(): string {
+    if (!this.ffaListing) { return '' }
+    return FileHelper.fileSizeString(this.ffaListing.size)
```
Comment (MelvinMallari) [inline src/components/ui/StaticFileMetadata.vue]: Can use ternary: `return !this.ffaListing ? '' : FileHelper.fileSizeString(this.ffaListing.size)`

File: src/components/ui/StaticFileMetadata.vue [hunk 2]
```diff
@@ -67,8 +69,14 @@ export default class StaticFileMetadata extends Vue {
...
+  }
+
+  public get costETH(): string {
+    if (!this.ffaListing) { return '' }
+    return FileHelper.costString(this.ffaListing.size, this.appModule.costPerByte)
```
Comment (MelvinMallari) [inline src/components/ui/StaticFileMetadata.vue]: `return !this.ffaListing ? '' : FileHelper.costString(this.ffaListing.size, this.appModule.costPerByte)`

File: src/util/FileHelper.ts [hunk 3]
```diff
@@ -79,6 +78,21 @@ export default class FileHelper {
     return `${fileSizeInBytes} bytes`
   }
 
+  public static costString(bytes: number, weiPerByte: number): string {
+    const costWei = weiPerByte * bytes
+    const costETH = costWei / 1000000000000000000
```
Comment (MelvinMallari) [inline src/util/FileHelper.ts]: We can use `web3`'s method to convert from wei_value to eth here: 

`web3.utils.fromWei(number [, unit])`

[doc](https://web3js.readthedocs.io/en/v1.2.0/web3-utils.html#fromwei)

Comment (ReidWilliams) [reply src/util/FileHelper.ts]: ok cool

**Pull request merged**